tests/integration_test.py

```diff
@@ -409,3 +410,14 @@ def test_invalid_feed(self, feed, fromversion):
         structure = mock_structure("", current)
         validator = IntegrationValidator(structure)
         assert not validator.is_valid_feed()
+
+    def test_is_fetch_params_exist(self):
```

Comment:
this is sanity
add more test for:
* missing param
* param not configured properly